addons/point_of_sale/static/src/app/navbar/navbar.xml

```diff
@@ -34,7 +34,24 @@
                                         Cash In/Out
                                     </a>
                                 </li>
+                                <li class="menu-item navbar-button" t-on-click="() => pos.toggleImages('product')">
+                                    <a class="dropdown-item py-2">
+                                        <t t-if="pos.show_product_images">Hide Product Images</t>
+                                        <t t-else="">Show Product Images</t>
+                                    </a>
+                                </li>
+                                <li class="menu-item navbar-button" t-on-click="() => pos.toggleImages('category')">
+                                    <a class="dropdown-item py-2">
+                                        <t t-if="pos.show_category_images">Hide Category Images</t>
+                                        <t t-else="">Show Category Images</t>
+                                    </a>
+                                </li>
```

Comment:
in master we don't want these options here. They are edited in res config settings